components/brave_new_tab_ui/brave_new_tab.html

```diff
@@ -20,6 +20,21 @@
   #root {
     height: 100%;
   }
+
+  ::-webkit-scrollbar {
+      width: 6px;
+  }
+
+  ::-webkit-scrollbar-thumb {
```

Comment:
Can you try

```css
body {
    scrollbar-color: var(--leo-color-gray-20) var(--leo-color-gray-40);
}
```

or something similar?
https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-color

Reply:
It's not quite the same as what the user encounters elsewhere in Brave. This is what the standard properties yield, as opposed to the prefixed options:
![image](https://github.com/brave/brave-core/assets/815158/0ac9752c-2e4b-469b-a215-4ff6b80a105a)

Reply:
Sweet, happy to accept the `::-webkit` stuff :smile: